tests/yamitranscode.cpp

```diff
@@ -244,11 +244,15 @@ SharedPtr<VppOutput> createOutput(TranscodeParams& para, const SharedPtr<VADispl
     return output;
 }
 
-SharedPtr<FrameAllocator> createAllocator(const SharedPtr<VppOutput>& output, const SharedPtr<VADisplay>& display)
+SharedPtr<FrameAllocator> createAllocator(const SharedPtr<VppOutput>& output, const SharedPtr<VADisplay>& display, int32_t poolsize)
 {
     uint32_t fourcc;
     int width, height;
-    SharedPtr<FrameAllocator> allocator(new PooledFrameAllocator(display, 5));
+
+    if(poolsize < 5){
```

Comment:
Better change to this:
`SharedPtr<FrameAllocator> allocator(new PooledFrameAllocator(display, std::max(extraSize, 5)));`